addon/kubeflow-operator/manifest.yaml

```diff
@@ -197,6 +197,32 @@ spec:
         name: manifests
         path: cert-manager/cert-manager-crds
     name: cert-manager-crds
+{{ if .Variables.OIDCProviderURL }}
```

Comment:
Do we also need to check `{{  if .Variables.DashboardURI }}` here? because I see that we use ` value: http://{{ .Variables.DashboardURI }}/login/oidc` few lines below, and what will happen, if the `DashboardURI` is empty?

Reply:
You are right, originally I had `DashboardURI` mandatory so I forgot to check